src/pages/en/install/auto.md

```diff
@@ -33,18 +33,60 @@ yarn create astro
+After confirming (y) to install `create-astro@latest`, you will be prompted to specify a project folder. (e.g. `./my-astro-site`)
 
-## 2. Install Dependencies
+### Choose a Starter Template
+You will then see a short list of starter templates to choose from: 
+- `Just the basics`: A great starter template for anyone wanting to explore Astro.
+- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
+- `Completely empty`: A template that just includes the bare minimium to get started.
+
+Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
+
+### Install Dependencies (optional)
+The wizard will offer to run an `install` command for you at this time, which is optional.
+
+> If you do not do so at this time, you will need to [install dependencies](/en/install/auto#2-install-dependencies) after the wizard has finished, before starting your project.
+
+### Install any Official Astro Integrations (optional)
+You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Turbolinks, Partytown, Sitemap) by running `astro add --yes`
+
+To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle selected/unselected. You can select multiple items at once, or you can continue without selecting any items.
+
+
+When you are satisfied with your selection, press return (enter) to submit. 
+
+>🟢 Selected integrations will display a solid (green) circle. (All integrations are **unselected by default**.) 
+>
+>⚠️ If every item is unselected, then no integrations will be added at this time. 
+
+
+These integrations, and any [Astro community integrations](https://astro.build/integrations), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide).
 
-When the `create-astro` install wizard is complete, you should see some recommended instructions on your screen to follow that will help you complete setup and start your new project. 
+### See Your Astro Config File
 
-The only required step remaining is to install your project's dependencies using a package manager like npm:
+After selecting your integrations to add, you should see the following terminal message followed by **the entire contents of the `astro.config.mjs` file** that Astro will include with your project:
+
+```bash
+Astro will make the following changes to your config file:
+```
+
+At this time, you can verify that any integrations have been added to your project configuation, as well as any required `install` actions have run.
```

Comment:
I’m not entirely sold on this section. At this point in following the guide, the CLI had logged the changes it had made to the config file, but is still asking me a question (about initializing a git repo).

This heading made me think I needed to maybe open the config file somehow. (After all that’s what you do with files.) I wonder if the heading is not needed and we could just inform the user that they’ll see any changes that the wizard has made logged to their terminal.

Reply:
(This ended up waaaay below, but I did make a change here.)